src/send/SendAmount/index.tsx

```diff
@@ -101,14 +108,20 @@ function formatWithMaxDecimals(value: BigNumber | null, decimals: number) {
 function SendAmount(props: Props) {
   const { t } = useTranslation()
   const dispatch = useDispatch()
+  const currencyPickerBottomSheetRef = useRef<BottomSheetRefType>(null)
+
+  const defaultToken = useSelector(defaultTokenToSendSelector)
```

Comment:
as mentioned in the PR description, most of these new lines are moved over from SendAmountHeader.tsx.